DALK.PL_ANALYZER/Models/Matches/Team.cs

```diff
@@ -7,39 +7,31 @@ namespace DALK.PL_ANALYZER.Models.Matches
 {
     public class Team
     {
-        private string name;
-
-        private string _url;
+        public string Name { get; set; }
+        public string URL { get; set; }
         public string IconUrl
         {
             get
             {
-                if (string.IsNullOrEmpty(_url))
+                if (string.IsNullOrEmpty(URL))
```

Comment:
I would go with check in SET instead of GET. This way get is 'cheaper' action.

Reply:
DONE.